### PR TITLE
WFLY-9754 an option to suppress transaction propagation for all beans

### PR DESCRIPTION
**Pull request by istudens**
Issue: https://issues.redhat.com/browse/EJBCLIENT-326

This PR supersedes https://github.com/wildfly/jboss-ejb-client/pull/358.

The conflicts of the original PR have been resolved and the code itself has been moved to a new 3.3 schema.

### CODE REVIEW

**Comment by istudens** [comment]
@fl4via @tadamski  could you review please? thanks

File: src/main/java/org/jboss/ejb/client/legacy/JBossEJBProperties.java [hunk 1]
```diff
@@ -80,6 +80,7 @@ public final class JBossEJBProperties implements Contextual<JBossEJBProperties>
     private static final String DEFAULT_ENDPOINT_NAME = "config-based-ejb-client-endpoint";
 
     private static final String PROPERTY_KEY_INVOCATION_TIMEOUT = "invocation.timeout";
+    private static final String PROPERTY_KEY_SUPPRESS_TX_PROPAGATION = "transaction.propagation.suppress";
```
Comment (jbaesner) [inline src/main/java/org/jboss/ejb/client/legacy/JBossEJBProperties.java:83]: This is the _legacy_ `JBossEJBProperties`, is it intended to add new configuration to the old _legacy_ configuration approach?

File: src/main/resources/schema/wildfly-client-ejb_3_1.xsd [hunk 2]
```diff
@@ -69,6 +69,21 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="tx-propagation-type">
```
Comment (jbaesner) [inline src/main/resources/schema/wildfly-client-ejb_3_1.xsd:72]: This block should be in `wildfly-client-ejb_3_3.xsd` only

**Comment by chengfang** [comment]
@istudens should this PR go toward `main` branch?

**Review by tadamski — CHANGES_REQUESTED** [review]
@istudens I also think that this PR is against the wrong branch. Could you please check?

**Review by tadamski — COMMENTED** [review]
closing after discussion with @istudens (out of date)

**Pull request closed without merging**